internal/endpoints/endpoints_db_test.go/endpoints_db_suite_test.go

```diff
@@ -0,0 +1,15 @@
+package endpoints_db_test
```

Comment:
nit pick: the parent directory of this file having `.go` at the end looks a little strange to me

Reply:
This is a good catch. i didn't notice it.

Reply:
Done! :100: